namui/src/namui/system/keyboard.rs

```diff
@@ -45,17 +45,8 @@ impl KeyboardSystem {
                         let mut pressing_code_set = pressing_code_set.write().unwrap();
                         pressing_code_set.insert(code);
 
-                        match code {
-                            Code::Space
```

Comment:
Does scrolling occur when the space bar is pressed?

Reply:
I'm not sure why I did care about scrolling. We don't have scroll-able element, isn't it?

Reply:
Yes, we only use the canvas that fits exactly the size of the viewport

Reply:
Then I think it's ok to merge

Reply:
I think the main purpose of #206 is alt+left or alt+right.